README.md

```diff
@@ -96,10 +94,26 @@ Copyright (c) 2012-2016 Nick Galbreath
 Licensed under the standard [BSD 3-Clause](http://opensource.org/licenses/BSD-3-Clause) open source
 license.  See [COPYING](/COPYING) for details.
 
+## BUILD TARGETS
+
+Some of the previous help runners have been merged into the Makefile. E.g.:
```

Comment:
Do we need to point this out?

Reply:
Well, I would start with this until we transition to completely removing them. I saw they were used here: https://github.com/libinjection/libinjection/pull/36, so maybe rethinking the documentation will be the best option in the end.